docs/community/com-tables/com-table-manage.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Manage tables
+layout: default
+parent: Community
+nav_order: 8
+---
+
+# How do I create and manage tables in FeatureBase Community?
+
+FeatureBase Community provides a UI which displays tables and their structure.
+
+Tables can be created:
+* using SQL queries
+* when importing data
+
+## Before you begin
+
+{% include /com-install/com-install-before-begin.md %}
+* [SQL statements](/docs/sql-guide/statements/statements-home)
+* [SQL create table](/docs/sql-guide/statements/statement-table-create)
+* [SQL alter table](/docs/sql-guide/statements/statement-table-alter)
+
+## Create tables by running SQL queries
```

Comment:
This is not all SQL queries

> Create tables

Reply:
Add "selected SQL queries" to clarify as you've mentioned